### PR TITLE
Add camera-streaming-daemon package

### PR DESCRIPTION
**Pull request by Rjasuja**
Enable Camera streaming daemon

### CODE REVIEW

**Comment by Rjasuja** [comment]
Installed librealsense(legacy branch for RS200) alongwith this package. Camera streaming daemon service launched successfully. Tested streaming using gstreamer pipelines as well

**Comment by anitha-suresh-zz** [comment]
Link to download librealsense legacy branch: https://github.com/IntelRealSense/librealsense/tree/legacy.
@rjasuja has tested with Legacy branch for now, as master branch had issues with RS200 sometime back. 
@Rjasuja , please try with master branch and update with the issues observed.

**Review by avinash-palleti — APPROVED** [review]

File: aero-camera-streaming-daemon/aero-camera-streaming-daemon_1.0/debian/control [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+Section: net
+Priority: optional
+Maintainer: Ritul Jasuja <ritul.jasuja@intel.com>
+Build-Depends: debhelper (>=9), dh-systemd (>= 1.5), dh-autoreconf, python-pip, git
+Standards-Version: 3.9.6
+Homepage: https://github.com/01org/camera-streaming-daemon
+
+Package: aero-camera-streaming-daemon
+Architecture: any
+Depends: ${shlibs:Depends}, ${misc:Depends}, python-future
```
Comment (anselmolsm) [inline aero-camera-streaming-daemon/aero-camera-streaming-daemon_1.0/debian/control]: Move python-future to Build-Depends

File: aero-camera-streaming-daemon/aero-camera-streaming-daemon_1.0/debian/control [hunk 2]
```diff
@@ -0,0 +1,13 @@
+Source: camera-streaming-daemon
+Section: net
+Priority: optional
+Maintainer: Ritul Jasuja <ritul.jasuja@intel.com>
+Build-Depends: debhelper (>=9), dh-systemd (>= 1.5), dh-autoreconf, python-pip, git
+Standards-Version: 3.9.6
```
Comment (anselmolsm) [inline aero-camera-streaming-daemon/aero-camera-streaming-daemon_1.0/debian/control]: See lintian output in the end of the package build.

File: aero-camera-streaming-daemon/aero-camera-streaming-daemon_1.0/debian/changelog [hunk 3]
```diff
@@ -0,0 +1,5 @@
+camera-streaming-daemon (1.0-1) unstable; urgency=medium
```
Comment (anselmolsm) [inline aero-camera-streaming-daemon/aero-camera-streaming-daemon_1.0/debian/changelog]: unstable -> xenial

File: aero-camera-streaming-daemon/aero-camera-streaming-daemon_1.0/debian/copyright [hunk 4]
```diff
@@ -0,0 +1,11 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: camera-streaming-daemon
+Source: https://github.com/01org/camera-streaming-daemon
+
+Files: *
+Copyright: 2016-2017 Intel Corporation
+License: Apache-2.0
+
```
Comment (anselmolsm) [inline aero-camera-streaming-daemon/aero-camera-streaming-daemon_1.0/debian/copyright]: Add:
```
 /usr/share/common-licenses/Apache-2.0
```

File: aero-camera-streaming-daemon/aero-camera-streaming-daemon_1.0/debian/copyright [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+Files: config/*
+Copyright: 2016 Intel Corporation
+License: Apache-2.0
```
Comment (anselmolsm) [inline aero-camera-streaming-daemon/aero-camera-streaming-daemon_1.0/debian/copyright]: Add:
```
 /usr/share/common-licenses/Apache-2.0
```

File: aero-camera-streaming-daemon/aero-camera-streaming-daemon_1.0/debian/aero-camera-streaming-daemon.install [hunk 6]
```diff
@@ -0,0 +1,2 @@
+config/main.conf etc/csd
+config/csd etc/init.d
```
Comment (anselmolsm) [inline aero-camera-streaming-daemon/aero-camera-streaming-daemon_1.0/debian/aero-camera-streaming-daemon.install]: Shouldn't be a systemd service?

**Review by anselmolsm — CHANGES_REQUESTED** [review]

**Comment by anselmolsm** [comment]
Also, please rebase

**Comment by anselmolsm** [comment]
Reverted. Please resubmit.

File: aero-camera-streaming-daemon/aero-camera-streaming-daemon_1.0/debian/control [hunk 7]
```diff
@@ -0,0 +1,12 @@
+Source: camera-streaming-daemon 
+Priority: optional
```
Comment (anselmolsm) [inline aero-camera-streaming-daemon/aero-camera-streaming-daemon_1.0/debian/control:2]: **Section** field is missing

**Pull request merged**